app/src/pages/Box.jsx

```diff
@@ -69,39 +69,25 @@ function App() {
         </div>
         <h4>Paddings</h4>
         <div className="test-case" id="padding-xxsmall">
-          <DxcBox size="medium" padding="xxsmall">
```

Comment:
Please remove the whole `Paddings` section.

Reply:
done!